docs/topics/proc_using-oauth2-with-reactive-http-web-service-webflux.adoc

```diff
@@ -0,0 +1,108 @@
+[id='using-oauth2-authentication-with-a-reactive-spring-boot-webflux-application_{context}']
+= Using OAuth2 authentication in a reactive {SpringBoot} application with WebFlux
+
+Set up OAuth2 for your {SpringBoot} Webflux application and authenticate with it using your client ID and client secret. 
+
+.Prerequisites
+
+* Java JDK 8 or Java JDK 11 installed
+* A GitHub account
+* Maven installed
+* A Maven-based application project link:{link-guide-spring-boot}#configuring-your-application-to-use-spring-boot_spring-boot[configured to use Spring Boot]
+
+
+.Procedure
+
+. link:https://github.com/settings/developers[Register a new OAuth 2 application] on your Github account.
+Ensure that you provide the following values in the registration form:
```

Comment:
I suggest either making this a single sentence with the above: Register a new OAuth 2 application on your Github account and provide the following values in the registration form:

or drop the "Ensure that you"